docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+---
+title: MUI X v6.0.0-alpha.0
+description: We’re kicking off the development of the next version.
+date: 2022-09-30T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['News']
+---
+
+We’re kicking off the development of MUI X v6. Both the Data Grid and Date Pickers are getting a new major version. We want to share our plans and invite you to join us on this next step.
+
+:::warning
+
+It’s important to notice that only MUI X is getting a new version; The core components remain in v5 for a while longer.
+You can rest assured that all MUI X v6 components will be fully compatible with MUI Core v5.
+::::
+
+## How is it going to work?
+
+Before officially releasing v6, we'll go through a few months with pre-releases. We’ll follow the current weekly release process and [semver versioning](https://semver.org/); The difference now is that instead of v5, we’ll ship v6 pre-release packages.
```

Comment:
We only use straight quotes for consistency

```suggestion
Before officially releasing v6, we'll go through a few months with pre-releases. We'll follow the current weekly release process and [semver versioning](https://semver.org/); The difference now is that instead of v5, we'll ship v6 pre-release packages.
```

There should be a way to add a rule with markdownlint about it.